### PR TITLE
fix(yay): fix -Si alignment when language is Chinese (#1799)

### PR DESCRIPTION
**Pull request by linweiyuan**
before fix:
image

after fix:
image

### CODE REVIEW

File: pkg/text/print.go [hunk 1]
```diff
@@ -90,7 +91,26 @@ func PrintInfoValue(key string, values ...string) {
...
+		language     = gotext.GetLanguage()
+		keyTextCount = keyLength - delimCount
+	)
+
+	if strings.HasPrefix(language, "zh") {
+		keyRuneCount := utf8.RuneCountInString(key)
+		switch {
+		case len(key) == keyRuneCount:
+		// these keys are all English words, do nothing, use the existing logic
+		case strings.HasSuffix(language, "TW") && key == "AUR 網址":
```
Comment (Jguer) [inline pkg/text/print.go]: This is liable to change and may be broken :frowning:

**Review by Jguer — CHANGES_REQUESTED** [review]
I think this approach might be error prone and go out of sync easily.

Alternative approaches:

- Go allows us to get the real length in bytes instead of in characters, maybe this can be used for a generic solution
https://www.golangprograms.com/golang-get-number-of-bytes-and-runes-in-a-string.html 
https://stackoverflow.com/questions/49397913/difference-between-length-of-rune-slice-and-runecountinstring

- or, I think you might be able to use https://siongui.github.io/2018/10/11/go-check-if-rune-value-is-chinese-character/ instead of grabbing the key, but then it will be only chinese symbolic specific (which for now seems to be ok)

File: pkg/text/print.go (hunk 1 above)
Comment (linweiyuan) [reply pkg/text/print.go]: You are right, let me update this.

**Comment by Jguer** [comment]
@linweiyuan if you're having issues with getting the linter running locally, I can handle touching up the linting before merging if that helps :+1:

**Review by Jguer — APPROVED** [review]
Thanks for your contribution and changing it up @linweiyuan :slightly_smiling_face: , I'm merging this

**Pull request merged**